### ISSUE TITLE
[suggestion] Icons package

### ISSUE THREAD

**Issue by azev** [erickutcher/httpdownloader#111]
Hey!

1st I'd like to complement the author for this wonderful program. Simple, effective and neat.
Very nice indeed!

i'd like to suggest new icons to the programs. there are a lot of `free icon packages` on the web.

Cheers!

**Comment by smaragdus** [comment]
As far as I know the icons of [HTTP Downloader](https://erickutcher.github.io/#HTTP_Downloader) are hand-made and therefore unique so the whole program has unique look and feel. I see no advantage the current icons to be replaced with some banal icon set.

**Comment by erickutcher** [comment]
The default program icon can't be changed (unless you use a resource editor or recompile the program yourself), but you can change the toolbar icons, tray icon, and URL drop window background.

In the same folder as the program, you can create the following files:

[main.ico](https://github.com/erickutcher/httpdownloader/raw/master/HTTP_Downloader/icon.ico)
[tray.ico](https://github.com/erickutcher/httpdownloader/raw/master/HTTP_Downloader/icon_tray.ico)
[toolbar.bmp](https://github.com/erickutcher/httpdownloader/raw/master/HTTP_Downloader/toolbar.bmp)
[drop.bmp](https://github.com/erickutcher/httpdownloader/raw/master/HTTP_Downloader/drop.bmp)

**Comment by azev** [comment]
Thanks Erich.
That information should be put in the `readme.txt` along with the command line options.
A `readme.txt` is always useful.

**Comment by Dhyfer1** [comment]
@erickutcher What software do you use to create bmp files with transparency? For example the pink color in the toolbar.bmp file is not visible when I open httpdownloader. So how do you do it?

**Comment by erickutcher** [comment]
The pink color is used as a transparent mask. The hex color value for it would be #FF00FF. If any pixel is colored that value, then it'll be transparent and the program's background will show through. The program doesn't support semi-transparent pixels though. Each icon in the toolbar.bmp file has a maximum width and height of 24 pixels.

You can use any drawing program to create the bitmap. Microsoft Paint is sufficient. You could use GIMP too if you have that.

**Comment by Dhyfer1** [comment]
> The pink color is used as a transparent mask. The hex color value for it would be #FF00FF. If any pixel is colored that value, then it'll be transparent and the program's background will show through. The program doesn't support semi-transparent pixels though. Each icon in the toolbar.bmp file has a maximum width and height of 24 pixels.
> 
> You can use any drawing program to create the bitmap. Microsoft Paint is sufficient. You could use GIMP too if you have that.

Sorry, I'm still lost. Let me see if I understood, for example for the toolbar you create a rectangle filled with the color #FF00FF, then you place the icons and finally save it as a bmp file? I was doing it in Paint (from Windows 10), but in Paint there is no option to create a mask. 😕

**Comment by erickutcher** [comment]
Yeah you can do it that way. Start with a completely pink rectangle and then position the icons.

**Comment by Dhyfer1** [comment]
> Yeah you can do it that way. Start with a completely pink rectangle and then position the icons.

Hey, thanks a lot. I did it the way I understood it and it works as expected. :) I was just testing it with the icons in the 7zip toolbar, since 7zip also uses bmp files for its icons.

By the way, those 4 files in your comment above where are they located? As httpdownloader is not an installer, it is a portable program, after unzipping HTTP_Downloader_64.zip I cannot find any of those 4 files

**Comment by erickutcher** [comment]
They're only in the repository. You can just download them from those links and save them into the same folder as the program.

**Comment by Dhyfer1** [comment]
> They're only in the repository. You can just download them from those links and save them into the same folder as the program.

I understand, it's clear, I must save them in the same httpdownloader folder. Ok
